CONTRIBUTING.md

```diff
@@ -25,10 +25,13 @@ Once this is complete end users who are using older releases of `kubernetes-base
 
 ## Addon Revisions
 
-You will find that any particular addon directory (e.g. `addons/prometheus`) may have several directories and several manifests nested in them each with variants of that addon. These are what we refered to above as "revisions".
+For Addons, "revisions" are a reference to the application version with an added revision count which indicates the latest iteration on that version. This enables multiple different versions of an Addon which ultimately utilize the same underlying application version so that configuration and other aspects of the Addon can change without overriding a previously released Addon.
 
-The **intention of revisions is to maintain a flat history of addon changes**. If you are making changes to any particular addon you should be making a revision of that addon as a copy of the original file with the changes made therein and the `addon-revision` version updated to reflect the new version appropriately.
+The [Kubeaddons Catalog API](https://github.com/mesosphere/kubeaddons) supports two different modes for addon repositories to host revisions:
```

Comment:
Do we really want to call this "Catalog API"? I'd prefer "Kubeaddons controller". I for myself think of github.com/mesosphere/kommander-catalog-api when I hear "Catalog API".

```suggestion
The [Kubeaddons controller](https://github.com/mesosphere/kubeaddons) supports two different modes for addon repositories to host revisions:
```

Reply:
The Kubeaddons Catalog API is the Go API at mesosphere/kubeaddons/pkg, so I generally think of this as correct :thinking: it's not specifically an aspect of the controller which this relates to.

Reply:
If you insist on the "Catalog" terminology here, would you mind changing the link to point to the specific Go API that you're referring to here?

Reply:
Sure thing
```suggestion
The [Kubeaddons Catalog API](https://github.com/mesosphere/kubeaddons/tree/master/pkg/catalog) supports two different modes for addon repositories to host revisions:
```